packages/theme-chalk/src/common/var.scss

```diff
@@ -996,9 +996,14 @@ $menu: map.merge(
     'bg-color': getCssVar('fill-color', 'blank'),
     'hover-bg-color': getCssVar('color-primary-light-9'),
     'item-height': 56px,
+    'sub-item-height': calc(#{getCssVar('menu-item-height')} - 6px),
+    'horizontal-sub-item-height': 36px,
     'item-font-size': getCssVar('font-size-base'),
     'item-hover-fill': getCssVar('color-primary-light-9'),
     'border-color': getCssVar('border-color'),
+    'top-padding': 20px,
```

Comment:
Also, should this only affect padding-left? It has strange consequences on the arrow otherwise.

0px
![image](https://user-images.githubusercontent.com/4075314/167648114-5c1a1ebf-a149-49c9-a0c8-653b04bb803f.png)

40px (notice the arrow is aligned differently from sub-menu)
![image](https://user-images.githubusercontent.com/4075314/167648270-a6d70aab-3cec-49f8-897b-f62a27ce230d.png)

Reply:
I'm seeing weird behavior with this, since every time an `el-menu` is used as a child, it resets all of these variables.

Level padding is ignored in this case:

```
.el-menu {
  --el-menu-top-padding: 0;
}
```

![image](https://user-images.githubusercontent.com/4075314/167654312-3a1cbcdd-efbf-4c0c-b217-18bfa0536ade.png)

Arrows are misaligned:

```
.el-menu:not(.el-menu .el-menu) {
  --el-menu-top-padding: 0;
}
```
![image](https://user-images.githubusercontent.com/4075314/167654693-679c7fa7-2f46-42e6-af97-f9590d285dc7.png)

Reply:
@YunYouJun Would it be better to declare this variable on :root instead of `.el-menu`, so it can be inherited by sub-menu instead of overwritten?

Reply:
> Level padding is ignored in this case:

`--el-menu-level-padding` defaults to `var(--el-menu-top-padding)`, hence the same value.

Reply:
> > Level padding is ignored in this case:
> 
> `--el-menu-level-padding` defaults to `var(--el-menu-top-padding)`, hence the same value.

I think it's just the naming of `el-menu-top-padding` that was throwing me off here, because it suggests only the level: 0 item is affected. But in reality it is affecting all the items. You're looking for a way to configure that level: 0 item differently from all of the sub-menu children. I'd reverse the variable dependency.

```
'level-padding': 20px,
'top-level-padding': getCssVar('level-padding'),
```

Reply:
I'll make them separate.

Reply:
> @YunYouJun Would it be better to declare this variable on :root instead of `.el-menu`, so it can be inherited by sub-menu instead of overwritten?

Yes, I think we can just make the necessary variables global. `:root` can also be placed in `menu.scss`. Because it only serves menu.